src/ipc-handlers.ts

```diff
@@ -99,6 +103,28 @@ export async function getInstalledApps( _event: IpcMainInvokeEvent ): Promise< I
 	};
 }
 
+export async function importSite(
+	event: IpcMainInvokeEvent,
+	{ id, backupFile }: { id: string; backupFile: BackupArchiveInfo }
+) {
+	const site = SiteServer.get( id );
+	if ( ! site ) {
+		throw new Error( 'Site not found.' );
```

Comment:
Note: New site creation will be handled from the UI.

Reply:
Not really from UI, but Studio would create an empty site just before it executes the import logic, correct?

Reply:
Yeah, by "from UI" I meant the renderer process ( the React part ).